apps/main/src/loan/components/PageLlamaMarkets/cells/LineGraphCell.tsx

```diff
@@ -39,29 +39,28 @@ type RateCellProps = {
 export const LineGraphCell = ({ market, type }: RateCellProps) => {
   const { snapshots, snapshotKey, isLoading, rate } = useSnapshots(market, type)
   const { design } = useTheme()
-  if (rate == null) {
-    return '-'
```

Comment:
sorry for the unrelated change. Just hiding the `-` as it's displaying already in the other column